catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsLoader.java

```diff
@@ -100,32 +102,39 @@ public static boolean enabled() {
    * <p>Caution should be exercised when executing code as system. Results from querying as system
    * are never returned to the client or cached. This means there is no risk of data leak.
    *
-   * @param framework the catalog framework, for creating system templates.
+   * @param catalogFramework the catalog framework, for creating system templates.
    * @param util for querying the catalog.
    * @param systemTemplates system templates loaded from config.
    */
   public static void bootstrap(
-      CatalogFramework framework, EndpointUtil util, List<Metacard> systemTemplates) {
-    Set<String> queryTitles =
-        executeAsSystem(() -> titlesTransform(util.getMetacardsByFilter(QUERY_TEMPLATE_TAG)));
-    Set<String> resultTitles =
-        executeAsSystem(() -> titlesTransform(util.getMetacardsByFilter(ATTRIBUTE_GROUP_TAG)));
+      CatalogFramework catalogFramework, EndpointUtil util, List<Metacard> systemTemplates) {
```

Comment:
✏️ If you're going to rename framework to catalogFramework I'd also rename util to endpointUtil

Reply:
done